cpp_api/call_bmc.cpp

```diff
@@ -0,0 +1,34 @@
+// Test file to try loading a GOTO-model into memory and running a sample verification run on it.
+#include <util/exception_utils.h>
+
+#include "api.h"
+#include "options.h"
+
+#include <iostream>
+#include <vector>
+
+int main(int argc, char *argv[])
```

Comment:
This testing command line interface should be in a separate directory from the rest of the implementation. I think we need to make it clear that it is not part of the API for user consumption.